sunbird-cassandra-utils/pom.xml

```diff
@@ -18,12 +18,12 @@
 		<dependency>
 			<groupId>org.cassandraunit</groupId>
 			<artifactId>cassandra-unit</artifactId>
-			<version>3.1.3.2</version>
+			<version>3.11.2.0</version>
 		</dependency>
 		<dependency>
 			<groupId>com.datastax.cassandra</groupId>
 			<artifactId>cassandra-driver-core</artifactId>
-			<version>3.1.0</version>
+			<version>3.7.0</version>
```

Comment:
Define version above and use that variable , so that changes will be only one place.

Reply:
Done